package.json

```diff
@@ -65,7 +92,7 @@
         "git add"
```

Comment:
in line above:
`"npm run lint-check",`
during last review I said that it accepts file/glob :V why isn't it changed

Thinking about precommit hook now I think we should delete this altogether, because if you commit from anywhere else but root it won't trigger, and personally I think it's a bit too flaky

Reply:
running tests && linters on CI should be enough and give us same result